03-FLEX/Portfolio/sergejs-vasjkvos.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Front-End Developer</title>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Vazirmatn:wght@300;500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style/style.css">
+</head>
+
+<body>
+  <header class="header">
+    <nav class="header-nav container">
+
+      <a class="header__logo" href="sergejs-vasjkvos.html">
+        <img width="70" height="50" src="img/logo.svg" alt="John Doe logo, home page button">
+      </a>
+
+      <ul class="header-list">
+        <li class="header-list__item"><a href="#about-me">About me</a></li>
+        <li class="header-list__item"><a href="#career">Skills</a></li>
+        <li class="header-list__item"><a href="#work-experience">Work experience</a></li>
+        <li class="header-list__item"><a href="#contact-me">Contact me</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <!-- About me -->
+    <section id="about-me" class="aboutme">
+      <div class="container aboutme-wrapper">
+        <img class="aboutme-photo" src="img/profile-foto.jpg" alt="Jonh Doe photo" width="200" height="200">
+
+        <div class="aboutme-info">
+          <p>Hello, my name is</p>
+          <h1 class="aboutme-info__name">John Doe</h1>
+          <span class="aboutme-info__position">Full Stack Developer</span>
+          <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dolores cum quae minus totam? Officia saepe quos
+            earum, at eum delectus voluptatibus perferendis voluptas consequuntur hic maxime explicabo est magnam
+            consectetur inventore. Aliquid quisquam quia accusantium similique neque labore.
+          </p>
+        </div>
+      </div>
+
+      <!-- <ul>
+        <li><a href="https://www.linkedin.com/" target="_blank">linkedin.com</a></li>
+        <li><a href="https://github.com/" target="_blank">github.com</a></li>
+        <li><a href="#" target="_blank">johndoe.com</a></li>
+      </ul> -->
+    </section>
+    <!-- /About me -->
+
+    <!-- Education -->
+    <section id="education" class="education container">
+      <h3 class="section-title">Education</h3>
+      <div class="education-wrapper">
+        <div class="education-item">
+          <h4>MSc in Computer Science</h4>
+          <p>University College London</p>
+          <p class="education-item__year">2015 - 2019</p>
+        </div>
+
+        <div class="education-item">
+          <h4>BSc Maths and Physics</h4>
+          <p>Imperial College London</p>
+          <p class="education-item__year">2007 - 2010</p>
+        </div>
+      </div>
+    </section>
+
+    <!-- /Education -->
+
+    <!-- Skills -->
+    <hr class="hr-line">
```

Comment:
Not sure whether you need to use classes in your case. You don't have different styles for hr' (at least if I didn't miss this), so you can stick with selecting hr by tags and styling them.